frontend/src/pages/landing/Landing.tsx

```diff
@@ -15,70 +15,79 @@ const STATUS_LABELS: { [key: string]: string } = {
+    {
+      fields: [],
+      component: KanbanCard,
+    },
+  ],
+  [
+    STATUS_LABELS.completed,
+    {
+      fields: [],
+      component: KanbanCard,
+    },
+  ],
+];
 
 interface LandingLoaderReturn {
   [key: string]: DestructionList[];
 }
 
 export const landingLoader = async (): Promise<LandingLoaderReturn> => {
   const lists = await listDestructionLists();
-  const statusMap: LandingLoaderReturn = {};
 
   // Initialize statusMap with empty arrays for each status
-  STATUSES.forEach((status) => {
-    statusMap[status] = [];
-  });
-
-  lists.forEach((list) => {
-    if (!statusMap[list.status]) {
-      statusMap[list.status] = [];
-    }
-    statusMap[list.status].push(list);
-  });
+  const statusMap = STATUSES.reduce((acc, val) => {
+    const status = val[0] || "";
+    const destructionLists = lists.filter(
+      (l) => STATUS_LABELS[l.status] === status,
+    );
+    return { ...acc, [status]: destructionLists };
+  }, {});
 
   return statusMap;
 };
 
 export const Landing = () => {
   const lists = useLoaderData() as LandingLoaderReturn;
 
-  const constructComponentList = (lists: DestructionList[]) => {
-    const constructAssigneeNames = (
-      assignees: DestructionList["assignees"],
-    ) => {
-      const sortedAssignees = assignees.sort((a, b) => a.order - b.order);
-      const getName = (assignee: DestructionList["assignees"][0]["user"]) =>
-        // If there's a first and last name, return the full name otherwise we return the username
-        assignee.firstName && assignee.lastName
-          ? `${assignee.firstName} ${assignee.lastName} (${assignee.role.name})`
-          : `${assignee.username} (${assignee.role.name})`;
-      return sortedAssignees.map((assignee) => getName(assignee.user));
-    };
-
-    return lists.map((list) => ({
-      id: list.name,
-      content: (
-        <KanbanCard
-          title={list.name}
-          days={timeAgo(list.created)}
-          assigneeNames={constructAssigneeNames(list.assignees)}
-          href={`/destruction-list/${list.name}`}
-          key={list.name}
-        />
-      ),
-    }));
+  const constructAssigneeNames = (assignees: DestructionList["assignees"]) => {
+    const sortedAssignees = assignees.sort((a, b) => a.order - b.order);
+    const getName = (assignee: DestructionList["assignees"][0]["user"]) =>
+      // If there's a first and last name, return the full name otherwise we return the username
+      assignee.firstName && assignee.lastName
+        ? `${assignee.firstName} ${assignee.lastName} (${assignee.role.name})`
+        : `${assignee.username} (${assignee.role.name})`;
+    return sortedAssignees.map((assignee) => getName(assignee.user));
   };
 
+  const objectLists = Object.values(lists).map((lists) =>
+    lists.map((list) => ({
+      key: list.name,
+      title: list.name,
+      days: timeAgo(list.created),
+      assigneeNames: constructAssigneeNames(list.assignees),
+      href: `/destruction-list/${list.name}`,
```

Comment:
Do we want to use the name here? In the detail view I had used the `pk`, but we should maybe add uuids to the destruction list model and then use those :thinking:

Reply:
Should be the PK indeed but the detail view is missing in main?

Reply:
Indeed: it is still a PR: https://github.com/maykinmedia/open-archiefbeheer/pull/54